src/marketing/recommendation-metrics.md

```diff
@@ -8,6 +8,8 @@ To analyze how well your recommendations are performing, Magento collects the fo
 
 -  **Impressions** - Number of times a Recommendation unit is loaded and rendered on a page. Note that the recommendation can be rendered on a page but not viewed by the shopper; for example, a Recommendation unit can be loaded “below the fold” of the browser’s viewport. In this case, the rendered unit would be counted as an impression, but a view would only be counted if the user scrolled the unit into view.
 
+-  **Viewable Impressions** - Number of Recommendation unit impressions that register a view.
```

Comment:
I guess a more specific definition would be "Number of Recommendation unit impressions that registered at least one view event."